src/app/static/src/app/store/translations/locales.ts

```diff
@@ -188,6 +211,52 @@ const en: Translations = {
     ART: "ART",
     apiCouldNotParseError: "Could not parse API response. Please contact support.",
     apiMissingError: "API response failed but did not contain any error information. Please contact support.",
+    axe: "Accessibility",
+    axeOnNaomiHeading: "Accessibility on Naomi",
+    axeOnNaomiParagraph1: "This statement applies to content published on <a href=\"https://naomi.unaids.org/\"> naomi.unaids.org </a>",
+    axeOnNaomiParagraph2: "We want as many people as possible to be able to use this website. For example, that means you should\n" +
+        "                    be able to:",
+    axeOnNaomiPurposeList: "<li>Resize your window with content being reformatted appropriately</li>\n" +
+        "                    <li>Adjust your text size without the site becoming less usable</li>\n" +
+        "                    <li>Read text easily due to sufficient contrast between foreground and background elements</li>",
+    axeOnNaomiParagraph3: "<a href=\"https://mcmw.abilitynet.org.uk/\">AbilityNet</a> has advice on making your device easier to\n" +
```

Comment:
It would be better if links opened on a new tab rather than navigating away from the page.